internal/kubernetes/crd.go

```diff
@@ -1,14 +1,16 @@
-package operator
+package kubernetes
 
 import (
 	"context"
 
-	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
-	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```

Comment:
a very minor nit: it'd be good to keep the more semantic import names (as we do in all other projects as well)

Reply:
See 6a56b4e